projects/observability/src/shared/graphql/request/builders/selections/observability-specification-builder.ts

```diff
@@ -213,14 +216,66 @@ export class ObservabilitySpecificationBuilder extends SpecificationBuilder {
     };
   }
 
+  public metricTimeseriesBandSpec(
+    metric: string,
+    aggregation: MetricAggregationType,
+    intervalDuration: TimeDuration
+  ): MetricTimeseriesBandSpecification {
+    const dateCoercer: DateCoercer = new DateCoercer();
+    const baselineSeriesAlias = `${aggregation}_baselineSeries_${intervalDuration.toString()}`;
+
+    // Timeseries does not require unique because it's deduping at the series level
+    const aggregationSelection = this.buildAggregationSelection(aggregation, false);
+
+    return {
+      ...this.getMetricSpecificationBase(metric, aggregation),
+      resultAlias: () => this.buildResultAlias(metric, aggregation, [intervalDuration.toString()]),
+      withNewIntervalDuration: newInterval => this.metricTimeseriesBandSpec(metric, aggregation, newInterval),
+      getIntervalDuration: () => intervalDuration,
+      asGraphQlSelections: () => ({
+        ...this.getMetricSelectionBase(metric),
+        children: [
+          {
+            path: 'baselineSeries',
+            alias: baselineSeriesAlias,
+            arguments: this.argBuilder.forIntervalArgs(intervalDuration),
+            children: [
+              { path: 'startTime' },
+              {
+                ...aggregationSelection,
+                children: [{ path: 'value' }, { path: 'upperBound' }, { path: 'lowerBound' }]
+              }
+            ]
+          }
+        ]
+      }),
+      extractFromServerData: resultContainer => {
+        const baselineSeries: GraphQlMetricBandInterval[] = resultContainer[metric][baselineSeriesAlias];
+
+        return baselineSeries.map(interval => {
+          const baselineArgInterval = interval[convertToGraphQlMetricAggregationPath(aggregation)]!;
+
+          return {
+            timestamp: dateCoercer.coerce(interval.startTime)!,
+            value: baselineArgInterval.value,
+            upperBound: baselineArgInterval.upperBound,
+            lowerBound: baselineArgInterval.lowerBound
+          };
+        });
+      }
+    };
+  }
+
   public buildAggregationSelection(
     aggregation: MetricAggregationType,
-    requireUnique: boolean = true
+    requireUnique: boolean = true,
+    fetchBounds: boolean = false
   ): GraphQlSelection {
     return {
       path: convertToGraphQlMetricAggregationPath(aggregation),
       alias: requireUnique ? aggregation : undefined,
-      arguments: this.argBuilder.forAggregationArgs(aggregation)
+      arguments: this.argBuilder.forAggregationArgs(aggregation),
+      children: fetchBounds ? [{ path: 'upperBound' }, { path: 'lowerBound' }] : []
```

Comment:
I don't remember why this doesn't include the `value` path for children. I believe it's only used in this file like:
```typescript
              {
                ...aggregationSelection,
                children: [{ path: 'value' }]
              }
```

Can we either 
- Remove the fetchBounds arg, and handle the bounds similar to value (which, I believe is what we're already doing) 
- Clean up this bit of tech debt by: making this method private, add the `value` child selection in here in either case and updating the callers.